controllers/providers/aws/autoscaling.go

```diff
@@ -54,17 +54,12 @@ func (a *AmazonClientSet) TerminateInstance(instance *autoscaling.Instance) erro
 	return nil
 }
 
-func (a *AmazonClientSet) SetInstanceStandBy(instance *autoscaling.Instance, scalingGroupName string) error {
-	instanceID := aws.StringValue(instance.InstanceId)
+func (a *AmazonClientSet) SetInstanceStandBy(instanceIDs []string, scalingGroupName string) error {
```

Comment:
can we rename to `SetInstancesStandBy` since we allow batching this now

Reply:
Done.